src/type_manipulation.cpp

```diff
@@ -0,0 +1,54 @@
+#include "type_manipulation.hpp"
+
+namespace RobotArm {
+char *TypeManipulation::stradd(char *dest, const char *src) {
```

Comment:
Why return a char? It's a pointer that is passed as a parameter, so the recipient already knows it.

Reply:
I also find it more logical to first mention the source and then the destination.

Reply:
ok Fair point.